lib/livebook_cli/server.ex

```diff
@@ -210,7 +213,11 @@ defmodule LivebookCLI.Server do
     autosave_path = Livebook.Config.autosave_path!("--autosave-path", path)
     opts_to_config(opts, [{:livebook, :autosave_path, autosave_path} | config])
   end
-
+  
+  defp opts_to_config([{:notebook, notebook_path} | opts], config) do
+    opts_to_config([{:open, true} | opts], [{:livebook, :root_path, File.cwd!() <> notebook_path} | config])
```

Comment:
Note that `:root_path` is the default path for the file system navigation component and we use it in multiple places, so we don't want to point it to the notebook everywhere. Also setting `open: true` will go to the homepage, but the expected behaviour would be to open the notebook page right away.

See the `open_from_options` function, we can extend it to handle the new option. For opening the notebook you can do the same as [here](https://github.com/livebook-dev/livebook/blob/b728d9deba9f7be66b0b4ff7cc5e18f55458b22f/lib/livebook_app.ex#L70). For now it will import the notebook (equivalent to "Fork"), but ideally we should "Open" it. We already track this in #927 and it can be a separate PR :)